core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -27,6 +27,7 @@ import scala.collection.JavaConverters._
 import scala.collection.Map
 import scala.collection.immutable
 import scala.collection.mutable.HashMap
+import scala.collection.mutable.HashSet
```

Comment:
It's for testing right?

Reply:
Yeah

Reply:
Expect this line to trigger compile error

Reply:
![image](https://user-images.githubusercontent.com/1475305/99769568-7f16c780-2b41-11eb-8b7f-c400e4b29602.png)
Looks like it's working in Scala 2.12

Reply:
![image](https://user-images.githubusercontent.com/1475305/99777005-a757f380-2b4c-11eb-9a44-b546478b5d40.png)
`-Wunused:imports` working in Scala 2.13